docs/implementers/enable-sso.md

```diff
@@ -1 +1,111 @@
-# <small>:construction:</small> Configuring Single Sign-On
\ No newline at end of file
+# Enabling and Disabling Single Sign-On (SSO)
+
+Ozone HIS provides support for Single Sign-On (SSO), enabling users to authenticate once and seamlessly access all integrated applications without repeated logins. SSO is powered by [Keycloak](https://www.keycloak.org), which serves as the central Identity Provider (IDP) for managing authentication and authorization throughout the Ozone ecosystem.
+
+With SSO enabled, all authentication requests are routed through Keycloak. It replaces each application's individual login page with a unified Keycloak login page for all users. This provides a consistent login experience across all integrated applications. For branding or user experience customization, you can modify the Keycloak login page, see [Customizing the Keycloak Login Page](../implementers/branding-apps.md#white-labelling-keycloak) for more details.
+
+## SSO Implementation per App
+
+Each application in Ozone HIS integrates with Keycloak using different mechanisms.
+
+!!! note
+    The `ozone-realm.json` file configures the Keycloak (IdP) by defining all client applications that will use Keycloak for authentication. Make sure each application (OpenMRS, Odoo, SENAITE) has a corresponding client entry in this file, including the correct redirect URIs and required settings.
+
+### OpenMRS (EMR)
+
+OpenMRS integrates with Keycloak for SSO using the [OpenMRS OAuth2 Login Module](https://github.com/openmrs/openmrs-module-oauth2login#openmrs-oauth-20-login-module). This module is included by default but is initially disabled, you can enable it by setting the `oauth2.enabled` property to true in the `oauth2.properties` file.
+
+!!! note
+     Enabling SSO in OpenMRS effectively disables basic authentication, so users will authenticate through the Keycloak login page rather than using traditional username and password prompts.
```

Comment:
```suggestion
     Enabling SSO in OpenMRS effectively disables basic authentication, so users will authenticate through the Keycloak login page rather than using traditional username and password prompts. No basic authentication will be allowed.
```